components/function-controller/deploy/function-base-image/nodejs12/kubeless-npm-install.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env sh
+
+set -e
+
+export HOME="/tmp"
+registry=${NPM_REGISTRY:-"https://registry.npmjs.org"}
+scope="${NPM_SCOPE:-""}"
+
+if [ -n "${scope}" ]; then
+  scope=${scope}:
+fi
+
+cd "$KUBELESS_INSTALL_VOLUME"
+npm config set "${scope}registry" "${registry}"
+
+if [ -n "${NPM_CONFIG_EXTRA}" ]; then
+  npm config set "${NPM_CONFIG_EXTRA}"
+fi
+
+npm install --production --prefix="$KUBELESS_INSTALL_VOLUME"
```

Comment:
I had to copy this file to adapt it to sh, original one from https://github.com/kubeless/runtimes/blob/master/stable/nodejs/kubeless-npm-install.sh works only with bash